src/sentry/eventstore/snuba/backend.py

```diff
@@ -62,7 +62,9 @@ def get_events(
         )
 
         if "error" not in result:
-            return [SnubaEvent(evt) for evt in result["data"]]
+            events = [SnubaEvent(evt) for evt in result["data"]]
+            self.bind_nodes(events)
```

Comment:
This way we would be doing one request to bigtable per event. I guess, it would be more efficient to rely on nodestore.get_multi that makes one query and streams the results.
Also would the performance improvement be still relevant if we fetch like 1000 events (which means blasting bigtable with 1000 requests)

Reply:
There were 2 place where we were previously fetching more than 100 events:
- unmerge: This fetches 500 events at a time, since it was already doing bind_nodes immediately after the eventstore fetch (which has been removed) there should be no difference in performance
- nodestore deletions: Rewrote to not use eventstore since we only need to get 2 columns

Reply:
If we are doing a multi_get (which we are) I think we should be fine with that. But we should keep an eye on this.

Reply:
The only place we are adding a multi get where where we weren't (at some later point outside eventstore) is in the group deletions task, we should keep an eye on that.